etc/links.profile

```diff
@@ -0,0 +1,51 @@
+# Firejail profile for elinks
+# Description: Text WWW browser
+# This file is overwritten after every install/update
+#
+# Persistent local customizations
+include links.local
+# Persistent global definitions
+# include globals.local
+
+blacklist /tmp/.X11-unix
+
+noblacklist ${HOME}/.links
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.links
+whitelist ${HOME}/.links
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+ipc-namespace
+machine-id
+netfilter
+no3d
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+tracelog
+
+disable-mnt
+private-cache
```

Comment:
Can you try also adding `private-bin links`? This works on my system (Arch).

Reply:
This only works for me if I am not using any associated programs and since the options are practically infinite here I'd put that as a comment:
```
# uncomment if you want to allow access only to user-configured associated programs
# private-bin links,sh,PROGRAM1,PROGRAM2,PROGRAM3
```

Reply:
Maybe better:
```
# if you want to use user-configured programs add 'private-bin PROGRAM1,PROGRAM2' to your links.local
# or append 'PROGRAM1,PROGRAM2' to this private-bin line
private-bin links,sh
```